lib/gitsh/tab_completion/tokens_to_words.rb

```diff
@@ -0,0 +1,47 @@
+module Gitsh
+  module TabCompletion
+    class TokensToWords
+      def self.call(tokens)
+        new(tokens).call
+      end
+
+      def initialize(tokens)
+        @tokens = tokens
+      end
+
+      def call
+        words
+      end
+
+      private
+
+      attr_reader :tokens
+
+      def words
+        token_groups.map do |tokens|
+          tokens.inject("") do |result, token|
+            if token.type == :WORD
```

Comment:
I see that this was an existing method that moved, but wondered would a `case` statement make sense here?

Reply:
Good call